recipe/utf_8_mac.patch

```diff
@@ -0,0 +1,1737 @@
+--- a/lib/Makefile.in	2022-02-12 07:13:33
++++ b/lib/Makefile.in	2022-05-31 03:37:07
+@@ -159,6 +159,7 @@
+     converters.h \
+       ascii.h \
+       utf8.h \
++      utf8mac.h \
+       ucs2.h \
+       ucs2be.h \
+       ucs2le.h \
```

Comment:
```suggestion
```
We can leave this out I think because AFAICT it's only used for creating distribution files.